examples/dataviews/widget.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>GeoJSON Layer with CARTO Tiles Example</title>
+
+  <!-- Custom CSS -->
+  <link rel="stylesheet" type="text/css" href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css">
+  <link rel="stylesheet" href="https://libs.cartocdn.com/airship-style/v2.1.1/airship.css">
+
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    #map {
+      width: 100vw;
+      height: 100vh;
+    }
+  </style>
+</head>
+
+<body>
+  <section id="map"></section>
+
+  <as-category-widget class="as-p--16" heading="Cities" description="Number of countries by continent"
+    style="position: absolute; right: 10px; top: 10px; height: 368px;">
+  </as-category-widget>
+
+  <!-- Map libraries-->
+  <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.js"></script>
+  <script src="https://unpkg.com/deck.gl@8/dist.min.js"></script>
+  <script src="https://libs.cartocdn.com/airship-components/v2.1.1/airship.js"></script>
```

Comment:
Same, v2.3 without specifying patch

Reply:
I'm gonna update the 'template.html'.file in this PR, to use it as a reference for examples